iscript/src/iscript/mac.py

```diff
@@ -1081,7 +1090,6 @@ def get_notarization_status_from_log(log_path):
                     semaphore,
                     run_command(
                         [
-                            "sudo",
```

Comment:
This might mean we no longer need sudo on dep boxes. (That may change again if/when we revisit having scriptworker claim multiple concurrent tasks.)

Reply:
Yes, that sounds correct to me.